watch.go

```diff
@@ -41,11 +47,21 @@ loop:
 		select {
 		case <-watcher.Events:
 			for _, a := range args {
-				if err := e.RunTask(context.Background(), Call{Task: a}); err != nil {
```

Comment:
As long as taskCancellers is modified concurrently, this is subject to data-race, and some tasks might (theoretically) be started _after_ ok returns `false`.